docs/app/pages/components/components-sections/dashboard/dashboard/snippets/aria.html

```diff
@@ -0,0 +1,12 @@
+...
+
+<button class="widget-grab-handle"
+    uxDashboardGrabHandle
+    uxDashboardWidgetDragHandle
+    uxGrabAriaLabel="Drag Handle"
+    [uxGrabStartAnnouncement]="grabStartAnnouncement(widget1)"
+    [uxGrabConfirmAnnouncement]="grabConfirmAnnouncement(widget1)">
+    <ux-icon name="pan"></ux-icon>
+</button>
```

Comment:
I tested the example code in the dashboard and it doesn't compile. Make sure to test code that you are suggesting for others to use!

https://plnkr.co/edit/9tY7B73YKXt0qr8D

Reply:
I'm not sure this has been resolved. Can you link the working plunker that you used to test this sample code?

Reply:
This should be it - https://plnkr.co/edit/B68rdWafJZpRO2kO

Maybe the way I have formatted this isn't the best. Its mainly I didn't want to add a lot of other unnecessary code unrelated to the accessibility feature. So I added the ellipsis to show that there is normally more code in-between.

Reply:
No worries, I see the misunderstanding now. The intent was to show how to bind a function to those properties to demo the ability to generate a dynamic value. This may not be obvious to consumers, so we should document an example of that functionality as well as just passing a string.

This one might be useful to demo for example:
![image](https://user-images.githubusercontent.com/26114101/140956334-0cd2e240-d34b-4529-89e5-2f10d93b29cd.png)

The ellipses aren't the problem, and just add noise to the sample, so all of that can be removed.

Reply:
Hi Donna, I may need a plunker example myself to write the demo file. I feel like this is simple but I have always encountered inputs that are just strings, booleans, etc. I understand the idea of binding the current context to the function but in this case I am a little stuck with the correct way to demo this. I may be over complicating this in my head!